apps/dashboard/src/components/workflow-cloud.tsx

```diff
@@ -0,0 +1,117 @@
+import { HTMLAttributes } from 'react';
+
+type WorkflowCloudProps = HTMLAttributes<HTMLOrSVGElement>;
+export const WorkflowCloud = (props: WorkflowCloudProps) => {
+  return (
+    <svg width="153" height="120" viewBox="0 0 153 120" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
did you run this through minification? https://jakearchibald.github.io/svgomg/

Reply:
I did not

Reply:
Neat tool!